### DISCUSSION TITLE
Open ScriptKit on Login?

### DISCUSSION THREAD

**Discussion by chancesm** [johnlindquist/kit#334] [Q&A] [answered]
Is it possible to open the app when I login to my Mac?

**Comment by chancesm** [comment, marked as answer]
I solved my own problem. Adding the app as a `Login Item` for my user in the `Users and Groups` section of the system preferences for my Mac solved this issue.

**Reply by johnlindquist** [reply]
The main menu under "Kit" has an "open at login" script (which does exactly what you found).

I'll  eventually need to build a settings dialog with some checkboxes though 😅
